### ISSUE TITLE
ToS for storage node

### ISSUE THREAD

**Issue by bwhm** [Joystream/storage-node-joystream#15]
Write Terms of Service for the storage node `Colossus`.

**Comment by bwhm** [comment]
The general ToS can be found in [here]. I'm not sure how you want to feature it, and if you want to add/remove anything, but discussions are welcome :)

**Comment by jfinkhaeuser** [comment]
Link is missing - sorry I didn't pick this up earlier. Is the idea that we link to the ToS somehow? How about during the signup process?

**Comment by bwhm** [comment]
> Link is missing 

Yeah, don't know what happened there!

I think it would suffice to have the node spit out this [URL](https://testnet.joystream.org/athens/pioneer/#/pages/tos) during signup process. 

For example, after `js_storage signup ...`, you get:
```
By singing up, you approve to our Terms of Service, found here: https://testnet.joystream.org/athens/pioneer/#/pages/tos. "Y/y" to confirm, any other button to escape.
```
Feel free to find a better and/or more articulate approach :)

**Comment by jfinkhaeuser** [comment]
Yeah, I'll do something like that... maybe a yes/no answer or something.
